### PR TITLE
Section: Factors [2]

### PR DESCRIPTION
**Pull request by olaassem**
Please review "factors.scss" for cleaner & less-bloated code.
Thanks!

### CODE REVIEW

File: src/stylesheets/layouts/_factors.scss [hunk 1]
```diff
@@ -1,22 +1,21 @@
 .key-factors {
-  @include flex;
   background-color: $color-mint;
-  margin-top: 1.5rem;
+  margin-top: 15px;
```
Comment (ten1seven) [inline src/stylesheets/layouts/_factors.scss]: If you already have a `rem` system (assuming you're using 62.5% to get base of 1rem = 10px), why move to pixels for layout?

File: src/stylesheets/layouts/_factors.scss [hunk 2]
```diff
@@ -146,8 +141,7 @@
     border: 5px solid $color-white;
     box-shadow: 0 0 0 5px rgba(#ffffff, 0.6);
     left: 0;
-    margin-left: auto;
-    margin-right: auto;
+    margin: auto;
```
Comment (ten1seven) [inline src/stylesheets/layouts/_factors.scss]: Are you safe setting margin top/bottom to `auto` as well? It's probably fine, but `margin: 0 auto;` is generally the way to be explicit (and safe) about margins so you know what you're getting.

**Review by ten1seven — CHANGES_REQUESTED** [review]
We can pick up the rem > px discussion in the comments.

File: src/stylesheets/base/_base.scss [hunk 3]
```diff
@@ -27,10 +27,10 @@ body {
...
   clip: rect(0 0 0 0);
-  height: 1px;
-  margin: -1px;
+  height: .1rem;
+  margin: -.1rem;
   overflow: hidden;
   padding: 0;
   position: absolute;
-  width: 1px;
+  width: .1rem;
```
Comment (ten1seven) [inline src/stylesheets/base/_base.scss:35]: Prettier is configured out of the box to add leading zeroes (so `0.1rem` versus `.1rem`). Have you looked into Prettier?

File: src/stylesheets/base/_typography.scss [hunk 4]
```diff
@@ -13,19 +13,19 @@
   @include text($secondary-font, normal, 1.8rem, $color-grey);
   text-align: center;
 
-  @media (min-width: 550px) {
+  @media (min-width: 34.4em) {
```
Comment (ten1seven) [inline src/stylesheets/base/_typography.scss]: I'd recommend not rounding so you can be pixel precise by using `34.375em` here.

**Review by ten1seven — APPROVED** [review]
I'd recommend checking all your media query px to em conversions and not round the decimal. Otherwise looks 👍

File: src/stylesheets/layouts/_factors.scss (hunk 1 above)
Comment (olaassem) [reply src/stylesheets/layouts/_factors.scss]: I wanted to streamline the units of measurement and (from my understanding 😐 ) pixels were recommended for vertical units. Thanks for the clarification!! Refactored:
- media queries --> em
- units --> rem
- line-heights --> unitless

File: src/stylesheets/base/_base.scss (hunk 3 above)
Comment (olaassem) [reply src/stylesheets/base/_base.scss:35]: I use Atom's Beautify. Will switch to Prettier if available in Atom!

Comment (ten1seven) [reply src/stylesheets/base/_base.scss:35]: https://atom.io/packages/prettier-atom

**Pull request merged**